src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,24 @@
+.dg_summary-list {
+  display: flex;
+  flex-wrap: nowrap;
+  color: $gray;
+  justify-content: flex-start;
+  align-items: flex-start;
+  padding: $margin-large;
+  margin-bottom: $default-margin;
+  .dg_summary-list__item {
+    p {
+      color: $gray;
+    }
+  }
+  h3 {
+    color: $black;
+  }
+  i {
+    display: block;
+    padding: 0 60px 0 0;
+    width: $default-padding;
+    align-content: center;
+    font-size: $h1-font-size;
```

Comment:
This is still not aligned center.

![image](https://user-images.githubusercontent.com/1933400/65887115-76769700-e36b-11e9-97a3-5fc7f0449316.png)

Can you please try this out, and let me know how it looks?

![image](https://user-images.githubusercontent.com/1933400/65887458-f3a20c00-e36b-11e9-9d38-768313201046.png)

Reply:
I tried but did not work . I am unable to center the icon